Support for default table properties (CSS: `.ck-content`) (#9555)
ckeditor/ckeditor5: packages/ckeditor5-table/docs/features/table.md

	.catch( ... );
```

### Default table styles

> The default tables feature allows configuring the default look of tables in the editor. The configuration object should be synchronized with the {@link builds/guides/integration/content-styles editor content styles}.

Reviewer: The beginning of this sentence seems to miss sth.

> The default tables feature allows configuring the default look of tables in the editor. The configuration object should be synchronized with the {@link builds/guides/integration/content-styles editor content styles}.

Reviewer: The beginning of this sentence seems to miss sth.

Author: Hm?

Reviewer: ```suggestion
The table styles feature allows configuring the default look of tables in the editor. The configuration object should be synchronized with the {@link builds/guides/integration/content-styles editor content styles}.
```

The **“Table properties”** button in the toolbar will show the applied table properties to the table.

The stylesheet for the editor displayed below are specified as follow:

```css
.ck-content .table {
    float: right;
    width: 500px;
    height: 250px;
}

.ck-content .table table {
    border-style: dashed;
    border-color: 'hsl(90, 75%, 60%)';
    border-width: 3px;
}
```

The same values must be passed to the editor configuration as the {@link module:table/tableproperties~TablePropertiesOptions `table.tableProperties.defaultProperties`} object:

```js
const tableConfig = {
    table: {
        tableProperties: {
            // The default styles for tables in the editor. They should be synchronized with the content styles.
            defaultProperties: {
	            borderStyle: 'dashed',
	            borderColor: 'hsl(90, 75%, 60%)',
	            borderWidth: '3px',
	            alignment: 'right',
	            width: '500px',
	            height: '250px'
            }
        }
    }
};
```

> The table element should be aligned to the `right` side by default. Its side should `500x250px`. Border style should be `dashed`, `3px` of its width, and color specified as `“Light green”`. The same touches new tables if they will be inserted into the editor

Reviewer: ```suggestion
The table element should be aligned to the `right` side by default. Its size should be `500x250px`. Border style should be `dashed`, `3px` of its width, and color specified as `“Light green”`. The same touches new tables if they will be inserted into the editor
```

Reviewer: > The same touches new tables 

I don't know what you mean here.

Author: ```suggestion
The table element should be aligned to the `right` side by default. Its size should be `500x250px`. Border style should be `dashed`, `3px` of its width, and color specified as `“Light green”`. The same will be applied for new tables if they will be inserted into the editor
```
